<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting5Widget</class>
 <widget class="QWidget" name="Setting5Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="CustomLabel" name="dnsServerLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="CustomLabel" name="gatewayLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="CustomLabel" name="subnetMaskLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="CustomLabel" name="IpLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="CustomLabel" name="ipAddressLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CustomLineEdit" name="ipLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="CustomLineEdit" name="subnetMaskLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="CustomLineEdit" name="gatewayLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="CustomLineEdit" name="dnsServerLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="CustomComboBox" name="ipAddressComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>CustomComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
